mbuild/utils/io.py

```diff
@@ -99,7 +99,20 @@ def import_(module):
     >>> # module from, etc) if the import fails
     >>> import tables
     >>> tables = import_('tables')
+
+    Notes
+    -----
+    The pybel/openbabel
     """
+    if module == 'pybel':
```

Comment:
It also might make sense to move this all to another function call, so the import code is less bloated.